plugins/pulp_rpm/plugins/distributors/export_distributor/generate_iso.py

```diff
@@ -87,6 +88,17 @@ def _make_iso(file_list, target_dir, output_dir, filename):
     if status != 0:
         log.error("Error creating iso %s; status code: %d; output: %s" % (file_path, status, out))
     else:
+        testFile = open(file_path, "rb")
+        h = hashlib.sha512()
+        while True:
+            piece = testFile.read(1024)
+            if piece:
+                h.update(piece)
```

Comment:
Since we want to calculate multiple digests, you should probably be iterating a list of hashers and updating all of them. 

Some examples from Pulp 3:
https://github.com/pulp/pulp/blob/master/pulpcore/pulpcore/app/files.py#L13
https://github.com/pulp/pulp/blob/master/pulpcore/pulpcore/app/files.py#L44